src/utilities/utilities.ts

```diff
@@ -100,13 +100,22 @@ export async function execSwift(
     return await execFile(swift, args, options, folderContext);
 }
 
+/**
+ * Get the file name of executable
+ *
+ * @param exe name of executable to return
+ */
+export function getExecutableName(exe: string): string {
```

Comment:
Is worthwhile combining this with `getSwiftExecutable` and `getSwiftExecutable` always appends the `.exe` to the end?

Reply:
I believe it already does that👀

This change belongs to #252 , if you have more questions let’s just move there

Reply:
Yeah just realised. Will move comment across